src/Features/CSharp/Portable/UseNamedArguments/CSharpUseNamedArgumentsCodeRefactoringProvider.cs

```diff
@@ -30,6 +33,22 @@ protected sealed override bool IsCloseParenOrComma(SyntaxToken token)
 
             protected override bool SupportsNonTrailingNamedArguments(ParseOptions options)
                 => ((CSharpParseOptions)options).LanguageVersion >= LanguageVersion.CSharp7_2;
+
+            protected override bool IsImplicitIndexOrRangeIndexer(ImmutableArray<IParameterSymbol> parameters, TSyntax argument, SemanticModel semanticModel)
+            {
+                var argType = semanticModel.GetTypeInfo(GetArgumentExpression(argument)).Type;
+                if (argType?.ContainingNamespace is { Name: nameof(System), ContainingNamespace: { IsGlobalNamespace: true } } &&
+                    (argType.Name == "Range" || argType.Name == "Index"))
```

Comment:
`nameof(Range)`?

Reply:
We're on .Net standard 2.0 still. We don't have Range